README.md

```diff
@@ -16,11 +16,12 @@ Liquibase also allows for automated rollback of SQL where possible, using markup
 formatted changesets.
 
 # Building
-This is a standard Gradle project with Jib integration, so can be built and tested with:
+This is a standard Gradle project with Jib integration, so can be built and tested with the following commands.
+Starting spanner emulator is required for tests
```

Comment:
The emulator is automatically started by the tests, so the user does not need to do that manually.